plugins/azure/resoto_plugin_azure/resource/compute.py

```diff
@@ -1734,6 +1767,9 @@ class AzureRestorePointCollection(AzureResource):
         access_path="value",
         expect_array=True,
     )
+    reference_kinds: ClassVar[ModelReference] = {
+        "successors": {"default": ["azure_resource"]},
```

Comment:
This type is very generic. Resoto will ensure no cycles in the graph (and this definition smells like cycles).
Can we reduce the type to something more specific? Restore points are valid for virtual machines, or?